### DISCUSSION TITLE
missing EF

### DISCUSSION THREAD

**Discussion by net4u** [Dotnet-Boxed/Templates#105] [Q&A]
hi,

I just create a new boilerplate project into an existing solution and I noticed that it miss EF. When I try to add the Nuget package it fails. Into other projects of solution (regular MVC by default VS template is no problem).
Also if I simply try to add Ado.Net Enitity Data Model it fails too.
![2016-06-16](https://cloud.githubusercontent.com/assets/255743/16112333/9abfd22a-33bf-11e6-92bd-02d56ae12a14.png)

Any hint please?

L.E.

The other projects that I talk about are build for some time. I noticed that if I try to make a new one it behave similar. I have to suppose that is something broke in my VS after some very recent updates

**Comment by RehanSaeed** [comment]
So you are just adding a NuGet package right? That error does not look very healthy and without knowing anythign else it looks like VS is messed up.

**Comment by net4u** [comment]
I just did a repair of VS and it's gone.
